code/index.html

```diff
@@ -1,30 +1,108 @@
+    <h6>LÄR DIG GÖRA KAFFE</h6>
+    <h1>Bli Barista</h1>
+    </div>
+  </header>
+
+  <section class="information">
     <!-- Signup form -->
 
+    <section class="sign-up">
+      <h3>Anmäl dig här</h3>
+      <form action="http://httpbin.org/anything" method="POST">
+        <label>
+          <input class="textinput" type="text" name="name" placeholder="YOUR NAME" required>
+        </label>
+        <label>
+          <input class="textinput" type="email" name="email" placeholder="EMAIL" required>
+        </label>
+        <label>
+          <input class="textinput" type="password" name="pwd" minlength="7" placeholder="PASSWORD" required>
+        </label>
+
+        <div class="radio-box">
+          <input type="radio" name="level" value="beginner">Beginner
+          <input type="radio" name="level" value="intermediate">Intermidiate
+          <input type="radio" name="level" value="advanced">Advanced
+        </div>
+
+        <div class="checkbox-box">
+          <label>
+            <input type="checkbox" name="check" value="Terms"> I accept the Terms & conditions
+          </label>
+          <label>
+            <input type="checkbox" name="check" value="Newsletter"> I want your monthly newsletter
+          </label>
+        </div>
+
+        <button type="submit">SUBMIT</button>
+      </form>
+    </section>
+
+
+    <!-- Accordion -->
+
     <section class="accordion">
-      <!-- Accordion -->
-      <h1>FAQ</h1>
 
-      <div class="question" id="section1">Section 1</div>
-      <div class="answer">
+      <h3>Frequently asked questions</h3>
+
+      <div class="question" id="section1">Fråga 1</div>
+      <div class="answer" id="a1">
         <p>Lorem ipsum...</p>
       </div>
 
-      <div class="question" id="section2">Section 2</div>
+      <div class="question" id="section2">Fråga 2</div>
       <div class="answer">
         <p>Lorem ipsum...</p>
       </div>
+
+      <div class="question" id="section3">Fråga 3</div>
+      <div class="answer">
+        <p>Lorem ipsum...</p>
+
     </section>
-    <!-- dont forget to link the javascript file here! -->
-  </body>
-</html>
+  </section>
+
+
+  <!-- <div onClick="toggle()" id="test">asdf</div> -->
+
+  <!-- dont forget to link the javascript file here! -->
+
+  <script src="script.js"></script>
+
+
+  <!--
+    <picture>
+      <source class="liten" srcset="images/Kaffe_liten.jpg" media="(max-width: 600px)" />
+      <source class="mellan" srcset="images/Kaffe_mellan.jpg" media="(max-width: 900px)" />
+      <img class="stor" src="images/kaffe_stor.jpg" alt="Kaffe" />
+    </picture>
+
+
+    <video controls autoplay>
+      <source src="videos/SampleVideo_360x240_1mb.mp4" media="(max-width: 600px)" type="video/mp4">
+      <source src="videos/SampleVideo_720x480_1mb.mp4" media="(max-width: 900px)" type="video/mp4">
+      <source src="videos/SampleVideo_1280x720_1mb.mp4" type="video/mp4">
+      <p>This browser doesn't support video...</p>
+    </video>
+  -->
+
```

Comment:
Code that we don't use can be removed before making a pull request. If you want to have it saved you can add it to a commit before removing it. Then it will be easy to access later on!